salt/master_utils/dns.sls

```diff
@@ -1,6 +1,3 @@
-include:
-  - master_utils.contrib
-
 create_dns_entry_for_salt_master:
   boto_route53.present:
     - name: {{ salt.pillar.get('salt_master:dns', 'salt.odl.mit.edu') }}
```

Comment:
I don't believe I see an entry for `salt_master:dns` in pillar. Am assuming there should be one at least for qa, no?

Reply:
It's defined here https://github.com/mitodl/salt-ops/blob/salt_masters/pillar/master/init.sls#L4